### PR TITLE
Unic Framework

### PR DESCRIPTION
**Pull request by rajkumardusad**
Unic web framework latest update

### CODE REVIEW

File: php/unic/composer.json [hunk 1]
```diff
@@ -1,26 +1,27 @@
...
+    "authors": [
+        {
+            "name": "Rajkumar Dusad",
+            "homepage": "https://github.com/rajkumardusad",
+            "role": "Developer"
+        }
+    ],
+    "require": {
+        "php": ">=5.6",
+        "unicframework/framework": "^1.0"
```
Comment (waghanza) [inline php/unic/composer.json:23]:
```suggestion
        "unicframework/framework": "~1.0.0"
```

File: php/unic/composer.json [hunk 2]
```diff
@@ -1,26 +1,27 @@
...
+    },
+    "minimum-stability": "dev",
+    "prefer-stable": true
```
Comment (waghanza) [inline php/unic/composer.json:26]: not sure it is required since it's the default behavior for **composer**

File: php/unic/composer.json [hunk 3]
```diff
@@ -1,26 +1,27 @@
...
+            "name": "Rajkumar Dusad",
+            "homepage": "https://github.com/rajkumardusad",
+            "role": "Developer"
+        }
+    ],
+    "require": {
+        "php": ">=5.6",
+        "unicframework/framework": "^1.0"
+    },
+    "minimum-stability": "dev",
```
Comment (waghanza) [inline php/unic/composer.json:25]: maybe better to leave the default 🤔 not sure what requires this

Comment (rajkumardusad) [reply php/unic/composer.json]: Done

File: php/unic/composer.json (hunk 2 above)
Comment (rajkumardusad) [reply php/unic/composer.json]: Done

File: php/unic/composer.json (hunk 1 above)
Comment (rajkumardusad) [reply php/unic/composer.json]: Done

**Review by waghanza — APPROVED** [review]

**Pull request merged**